src/compute-pr-actions.ts

```diff
@@ -78,8 +74,6 @@ function createEmptyActions(): Actions {
         responseComments: [],
         shouldClose: false,
         shouldMerge: false,
-        shouldUpdateLabels: false,
-        shouldUpdateProjectColumn: false,
```

Comment:
No time to look too deeply into this, but: doesn't this change the behavior of the two uses of `createEmptyActions`?

Reply:
You're right, thanks for catching that! I've reverted this bit :heavy_check_mark: